src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -618,7 +618,6 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
 
                         if (useTE)
                         {
-                            request.Content.Headers.ContentLength = 0;
```

Comment:
Why are you removing this?  Is it because you expect the HTTP stack to do this itself?

Reply:
Yes.  Also it's wrong.  To remove the ContentLength, we should set it to null.  But we don't need to do that, the handler will do it for us.